Solutions/Menes.Specs/Steps/OpenApiDefaultParameterParsingSteps.cs

```diff
@@ -115,9 +115,15 @@ public async System.Threading.Tasks.Task WhenITryToParseTheDefaultValueAsync()
 
             var context = new DefaultHttpContext();
 
+            this.parameters = await builder.BuildParametersAsync(context.Request, operationPathTemplate!);
+        }
+
+        [When(@"I try to parse the default value and expect an error")]
```

Comment:
We try and get rid of the `@` in these attributes where they aren't needed. If you've got Roslynator installed, it'll clean up all of these for you in one fix operation.

Reply:
Fixed